src/Writer/MarkdownWriter.php

```diff
@@ -94,7 +94,7 @@ protected function createPackageLockedLine(
         PackageManagerPackageInterface $package,
         DependencyData $dependency
     ): string {
-        $line = "##### {$package->getName()} `{$package->getVersion()}` {$dependency->getLockSymbol()}";
+        $line = "# {$package->getName()} `{$package->getVersion()}` {$dependency->getLockSymbol()}";
```

Comment:
This seem's to bei wrong, nor?
In `createPackageLine` below you're using **h2**?

Reply:
Excuse this fail of a PR. I thought I could fix this real quick in the GitHub online editor without checking out the branch :man_facepalming: :cake: will fix this!